aikau/src/main/resources/alfresco/menus/AlfFilteringMenuItem.js

```diff
@@ -44,9 +44,9 @@ define(["dojo/_base/declare",
        * @instance
        */
       postCreate: function alfresco_menus_AlfMenuItem__postCreate() {
-         if (this.filterTopic != null)
+         if (this.filterTopic)
          {
-            this.alfSubscribe(this.filterTopic, lang.hitch(this, "filter"));
+            this.alfSubscribe(this.filterTopic, lang.hitch(this, this.filter));
          }
          this.inherited(arguments);
```

Comment:
___[Comment only]___ It'd be more concise to just have:
`this.filterTopic && this.alfSubscribe(this.filterTopic, lang.hitch(this, this.filter));`